docker/local/common/ses-relay.py

```
@@ -0,0 +1,89 @@
+        f"To: {', '.join(to)}",
+        f"Subject: {subject}",
+        "MIME-Version: 1.0",
+        f"Content-Type: {content_type}",
+        "",
+        content,
+    ])
+    return raw, source, to
+
+
+def relay_pending() -> None:
+    req = urllib.request.urlopen(f"{LOCALSTACK_URL}/_aws/ses", timeout=5)
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)